storage/collector.go

```diff
@@ -50,6 +53,118 @@ func InsertCollector(newCollector Collector) error {
 	return nil
 }
 
+func setIndexesCollector(collectorC *mongo.Collection) error {
+	opts := options.CreateIndexes().SetMaxTime(10 * time.Second)
+	indexes := []mongo.IndexModel{
+		{
+			Keys:    bsonx.Doc{{Key: "path", Value: bsonx.Int32(1)}},
+			Options: options.Index().SetUnique(true),
+		},
+		{
+			Keys:    bsonx.Doc{{Key: "id", Value: bsonx.Int32(1)}},
+			Options: options.Index().SetUnique(true),
+		},
+	}
+
+	_, err := collectorC.Indexes().CreateMany(context.Background(), indexes, opts)
+	if err != nil {
+		return fmt.Errorf("create index error: %q", err)
+	}
+
+	return nil
+}
+
+// GetCollectors return all collectors in the database
+func GetCollectors() ([]byte, error) {
+	var collectors []Collector
+
+	client, err := conect()
```

Comment:
A conexão com o banco de dados não deve ser refeita a cada conexão, deve ser mantida. Citando o [tutorial oficial do driver](https://www.mongodb.com/blog/post/mongodb-go-driver-tutorial):

> It is best practice to keep a client that is connected to MongoDB around so that the application can make use of connection pooling - you don't want to open and close a connection for each query. However, if your application no longer requires a connection, the connection can be closed with client.Disconnect() like so:

Dessa forma você vai precisar criar uma classe (um client ou service) que guarde a conexão ou o DB+Columns (olhando o dadosjusbr tem exemplos disso).

Reply:
Certo. Vou criar uma issue para isso e começo a estudar essa configuração amanhã.